package-scripts.js

```diff
@@ -76,35 +64,6 @@ module.exports = {
   },
 }
 
-function concurrent(scripts) {
-  const {
-    colors,
-    scripts: quotedScripts,
-    names,
-  } = Object.keys(scripts).reduce(reduceScripts, {
-    colors: [],
-    scripts: [],
-    names: [],
-  })
-  const flags = [
-    //   https://github.com/kimmobrunfeldt/concurrently/issues/91
```

Comment:
This was fixed with a new flag `--kill-others-on-fail`, but you don't have to worry about that, it's all taken care of for you. In fact, you don't even need to install `concurrently` yourself! Just install `nps-utils` and it'll magically work :)